### DISCUSSION TITLE
求指路，如何实现文档中那种三维立体效果图的

### DISCUSSION THREAD

**Discussion by pangxiaobin** [vbenjs/vue-vben-admin#4399] [General]
请教下这种效果图是如何制作的，是ps吗，有没简单快速制作的方式。
<img width="835" alt="image" src="https://github.com/user-attachments/assets/2dbd9017-a5dc-491e-b060-8089a9b8dc3f">

**Comment by anncwb** [comment]
找UI设计

**Reply by pangxiaobin** [reply]
> 找UI设计

想找找看有没有在线的快速生成类似的，自己项目，没有UI。

**Comment by LiYuzZ** [comment]
旋转好角度，复制一个反转加线性渐变，纯css应该可以实现
